[2022-08-03 15:45:19,827] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: csv_dag.link_dag scheduled__2022-08-02T00:00:00+00:00 [queued]>
[2022-08-03 15:45:19,853] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: csv_dag.link_dag scheduled__2022-08-02T00:00:00+00:00 [queued]>
[2022-08-03 15:45:19,854] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-03 15:45:19,855] {taskinstance.py:1377} INFO - Starting attempt 1 of 2
[2022-08-03 15:45:19,855] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-03 15:45:19,911] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): link_dag> on 2022-08-02 00:00:00+00:00
[2022-08-03 15:45:19,932] {standard_task_runner.py:52} INFO - Started process 22715 to run task
[2022-08-03 15:45:19,963] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'csv_dag', 'link_dag', 'scheduled__2022-08-02T00:00:00+00:00', '--job-id', '116', '--raw', '--subdir', 'DAGS_FOLDER/testDag.py', '--cfg-path', '/tmp/tmpt6yimimw', '--error-file', '/tmp/tmpz3tqmnoj']
[2022-08-03 15:45:19,973] {standard_task_runner.py:80} INFO - Job 116: Subtask link_dag
[2022-08-03 15:45:20,224] {task_command.py:371} INFO - Running <TaskInstance: csv_dag.link_dag scheduled__2022-08-02T00:00:00+00:00 [running]> on host 36e1df4dd6ce
[2022-08-03 15:45:20,431] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=csv_dag
AIRFLOW_CTX_TASK_ID=link_dag
AIRFLOW_CTX_EXECUTION_DATE=2022-08-02T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-08-02T00:00:00+00:00
[2022-08-03 15:46:02,501] {python.py:173} INFO - Done. Returned value was: None
[2022-08-03 15:46:02,517] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=csv_dag, task_id=link_dag, execution_date=20220802T000000, start_date=20220803T154519, end_date=20220803T154602
[2022-08-03 15:46:02,552] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-03 15:46:02,595] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
